memory/ndctl.py

```diff
@@ -173,12 +173,29 @@ def disable_namespace(self, namespace='all', region='', bus='',
             args = '%s -r %s' % (args, region)
         if bus:
             args = '%s -b %s' % (args, bus)
-        if force:
-            args = '%s -f' % args
+        if verbose:
```

Comment:
any specfic reason to move force to verbose please have that part to commit mesage

Reply:
That is the correction we are doing here as it does not have a force option. Will have that as part of commit message

Reply:
:+1: